test/runner.js

```diff
@@ -14,63 +14,65 @@ test('returns new instance of runner without "new"', function (t) {
 test('runner.addTest adds a new test', function (t) {
 	var runner = new Runner();
 	runner.addTest(mockTitle, noop);
-	t.is(runner.stats.testCount, 1);
-	t.is(runner.tests.concurrent.length, 1);
-	t.true(runner.tests.concurrent[0] instanceof Test);
+	t.is(runner.tests.length, 1);
+	t.true(runner.tests[0] instanceof Test);
+	t.false(runner.tests[0].serial);
```

Comment:
Only use `t.true/false` when they're actually `true`/`false`. I know `tap` doesn't enforce this, but AVA does, so it will fail when we eventually move to self testing. Instead use `t.ok/notOk` to check truthy/falsy.

I just cleaned this up yesterday: https://github.com/sindresorhus/ava/commit/21885b8c7dd929a7496ea9105196dd2d67d02db5

// @vdemedes fyi